bootstrap-salt.sh

```diff
@@ -2427,9 +2427,9 @@ __create_virtualenv() {
     if [ ! -d "$_VIRTUALENV_DIR" ]; then
         echoinfo "Creating virtualenv ${_VIRTUALENV_DIR}"
         if [ $_PIP_ALL -eq $BS_TRUE ]; then
-            virtualenv --no-site-packages "${_VIRTUALENV_DIR}" || return 1
+            /usr/local/bin/virtualenv --no-site-packages "${_VIRTUALENV_DIR}" || return 1
```

Comment:
why are we hard coding the path here?

Reply:
Because this is the binary installed by `pip` explicitly the script. This eliminates possibility of running with distro-provided `virtualenv`, depending on `PATH` settings.

Reply:
ahhh make sense. thanks for the explanation.

Reply:
I found that if I run this script at the end of ubuntu installation (preseed late_command hook) via `sh` it could not resolve `virtualenv` without the full path